lib/api/bucketDeleteTagging.js

```diff
@@ -0,0 +1,50 @@
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const { pushMetric } = require('../utapi/utilities');
+const metadata = require('../metadata/wrapper');
+const util = require('node:util');
+
+/**
+ * Bucket Delete Tagging - Delete a bucket's Tagging
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+async function bucketDeleteTagging(authInfo, request, log, callback) {
+    log.debug('processing request', { method: 'bucketDeleteTagging' });
```

Comment:
This log may be more useful if also logging some request info, at least the bucket name and maybe also the owner ID

Reply:
done 684e92cd5c1651787050c6a7bb8e8f440cb8ce2f